internal/common/transformer.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// SecurityContext is a transformer function that removes runAsUser and runAsGroup from container's security context
+func SecurityContext(object *unstructured.Unstructured) error {
```

Comment:
Style nit - be specific when naming things and writing godoc:

```suggestion
// RemoveRunAsUserRunAsGroup is a Manifestival transformer function that removes runAsUser and runAsGroup from a Deployment container's security context
func SecurityContext(object *unstructured.Unstructured) error {
```